tests/conftest.py

```diff
@@ -113,6 +113,43 @@ def batch_id(self):
                '08f5308af03fd4aa18ff1d868f043b12dd7b0a792e141f000a2505acd4b7a956'
 
 
+class PublicKeyInformation:
+    """
+    Impose public key information data transfer object.
+    """
+
+    @property
+    def data(self):
+        """
+        Get data of public key information.
```

Comment:
Why not just `Get public key information`?

Reply:
@dmytrostriletskyi, done.